src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -9,6 +9,8 @@
 
 DEFINE_int32(default_parts_num, 100, "The default number of parts when a space is created");
 DEFINE_int32(default_replica_factor, 1, "The default replica factor when a space is created");
+DEFINE_int32(default_space_sum, 100, "The default number of space can be created");
+DEFINE_int32(default_max_replica_factor, 1, "The default max replica factor that a space can take");
```

Comment:
Please revise the naming of these two flags. _**default**_ is the value we take when user doesn't provide. You could infer from the fact how `default_parts_num` are used.

Reply:
good point, I changed these two flag names